### DISCUSSION TITLE
How to disable the mapping for `C-e`?

### DISCUSSION THREAD

**Discussion by gengdz** [saghen/blink.cmp#320] [Q&A] [answered]
The current configuration is as follows(Partial code):

```lua
{
    "saghen/blink.cmp",
    opts = {
      keymap = {
        preset = "enter",
        ["<Tab>"] = {
          LazyVim.cmp.map({ "snippet_forward", "ai_accept" }),
          "fallback",
        },
      },
    },
  }

```

I want to disable the `[C-e]` mapping defined in `preset = "enter"`, how should I do that?

**Comment by saghen** [comment, marked as answer]
I've just pushed a change that will ignore any keymap with no commands so you could do:  `['<C-e>'] = {}`. If you're not running on main, you could temporarily do `['<C-e>'] = { 'fallback' }` which effectively does the same thing

**Reply by gengdz** [reply]
Thank you very much, it works!
